sections/main-cart-items.liquid

```diff
@@ -116,48 +116,56 @@
                     </p>
```

Comment:
For the `cart-item__error`, sorry I should have mentioned, it's happening to me on Safari.